exporters/cr/exporter.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
```suggestion
	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Minor but we usually use this alias.